File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/cardview/CardViewActivity.java
package com.jdroid.android.sample.ui.cardview;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class CardViewActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return CardViewFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/cardview/CardViewActivity.kt
package com.jdroid.android.sample.ui.cardview

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class CardViewActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return CardViewFragment::class.java
    }
}